test/regression/external/notify-callback.spec.js

```diff
@@ -0,0 +1,25 @@
+const { getLastNotifications, simulateNotifyCallback } = require('../shared/helpers/notifications');
+const { sleepFor } = require('../shared/helpers/utils');
+const config = require('./config');
+
+/* eslint-disable no-undef */
+describe('notify callback attempt', function () {
+  let scheduledNotification, scheduledNotificationAfterCallback, scheduledNotificationNotifyId;
+  before(async () => {
+    await sleepFor(1000);
+    scheduledNotification = await getLastNotifications(config.baseUrl, config.userEmails.notifyCallbackTestEmail);
+    console.log(scheduledNotification);
```

Comment:
Did you mean to leave this in?

Reply:
🙊 oops. No, I will remove this. Thank you.